x-pack/plugins/security/public/plugin.tsx

```diff
@@ -240,3 +240,19 @@ export interface SecurityPluginStart extends SecurityPluginStartWithoutDeprecate
    */
   uiApi: UiApi;
 }
+
+function buildManagementServiceUiConfigs(config: ConfigType): ManagementAppConfigType | undefined {
+  let result;
+  const ui = config.ui;
+  const roleManagementEnabled = config.roleManagementEnabled;
+
+  if (ui) {
+    result = { ...ui };
+  }
+
+  if (roleManagementEnabled) {
+    result = { ...result, roleManagementEnabled };
+  }
+
+  return result;
+}
```

Comment:
__optional nit:__ might be a little bit simpler with something like this, but up to you:
```ts
  const uiConfig = config.ui ?? undefined;
  return config.roleManagementEnabled
    ? { ...uiConfig, roleManagementEnabled: config.roleManagementEnabled }
    : uiConfig;
```

Reply:
Method removed